docs/access-api-spec.md

```diff
@@ -626,6 +628,38 @@ The event results are grouped by block, with each group specifying a block ID an
   ```
 </details>
 
+---
+
+### Chain ID
+
+ChainID helps identify the Flow network (mainnet versus testnet). The following method can be used to query for the Chain ID
```

Comment:
do you think it add values if we also report `uptime`? I could extend the GetChainID method to report more details about the network.

Reply:
Yeah, I was thinking it might be worthwhile to generalize this a bit. 

Maybe `GetNetworkParameters`?

`chain_id` is just one of the parameters IMO. We might want to return information about the epoch, gas prices, etc in the future.

```
message GetNetworkParametersResponse {
    string chain_id = 1;
}
```

Reply:
how about `GetChainDetails`?